### ISSUE TITLE
Typescript error: Variable declaration expected

### ISSUE THREAD

**Issue by fernandopasik** [keithamus/sort-package-json#180]
After the release of 1.42.0 I'm getting this typescript error

```
node_modules/sort-package-json/index.d.ts:14:7 - error TS1134: Variable declaration expected.

14   var default: typeof sortPackageJsonCore;
         ~~~~~~~

node_modules/sort-package-json/index.d.ts:14:14 - error TS1134: Variable declaration expected.

14   var default: typeof sortPackageJsonCore;
                ~

node_modules/sort-package-json/index.d.ts:14:16 - error TS1134: Variable declaration expected.

14   var default: typeof sortPackageJsonCore;
                  ~~~~~~


Found 3 errors.
```

**Comment by fernandopasik** [comment]
would this one be fixed by #173 ?

**Comment by fernandopasik** [comment]
This was fixed by #189
